pytest_playwright/pytest_playwright.py

```diff
@@ -236,6 +236,9 @@ def context(
             context.tracing.stop()
 
     screenshot_option = pytestconfig.getoption("--screenshot")
+    screenshot_fullscreen = (
+        True if pytestconfig.getoption("--full-page-screenshot") == "on" else False
```

Comment:
the condition here already evaluates to the boolean, the `if` logic is unnecessary

Reply:
`if` condition has been removed when updating to `true_store`